modules/w25qxx/README.md

```diff
@@ -0,0 +1,114 @@
+# W25QXXX - SPI Flash Memory
+
+## Connections:
+
+1. Connect the W25QXX module to the SPI pins
+2. Ensure that all the headers for the SPI pins have been connected
+
+## Usage:
+
+1. Connect the development board to your computer using the USB cable
+2. Install the `SPIMemory` library from the library manager
+2. Upload `w25qxx.ino` to the board, using the Arduino IDE
+3. Open the Serial Monitor in the IDE, and set the baud rate to 115200
+4. Enter <pre>w <i>string</i></pre> into the terminal, to save the string
+into flash memory
+5. Once confirmation appears on the Serial console, disconnect the board and power it off
+6. Since flash memory is secondary memory, the data should remain on the board
+7. Connect the board once again, and open Serial Monitor
+8. Enter `r` in the monitor, to request the development board to read the data stored
+9. Verify that the data retrieved matches the data that was entered
+
+## Code Explanation
```

Comment:
The section below could have a title as it is not "code explanation" but more of "guidelines" or "precautionary notes". A better name would definitely help. Also, might want to wrap it up in `blockquotes`